understanding/change-on-request.html

```diff
@@ -179,43 +152,38 @@ <h2>Benefits</h2>
                   </li>
                   
                   
-                  <li>Some individuals with low vision, with reading and intellectual disabilities, and
-                     who have difficulty interpreting visual cues may benefit from additional cues in order
-                     to detect changes of context.
+                  <li>ロービジョンの利用者、読字及び知的障害のある利用者、そして視覚的な手がかりを解釈しづらい利用者は、コンテンツ制作者が手がかりを追加することによって、コンテキストの変化に気づくようになる。
                      
                   </li>
                   
                   
-                  <li>People with certain 
-                     <strong>cognitive limitations</strong> do not get confused if automatic redirects are performed by the Web server instead
-                     of the browser.
+                  <li>ブラウザの代わりにウェブサーバーによって自動的なリダイレクトが実行されると、特定の
+                     <strong>認知制限</strong>を持つ人が混乱しないですむ。
                      
                   </li>
                   
                   
                </ul>
             </section>
             <section id="examples">
-               <h2>Examples</h2>
+               <h2>事例</h2>
                <dl>
                   
-                  <dt>an "update now" button</dt>
+                  <dt>「今、更新する」ボタン</dt>
```

Comment:
「今、更新する」というラベルのボタンは見たことがありません。:-)
日本語で一般的なものというと「今すぐ更新」というくらいではないでしょうか。

```suggestion
                  <dt>「今すぐ更新」ボタン</dt>
```